algorithms/sort/bubble_sort.py

```diff
@@ -27,6 +28,7 @@ def swap(i, j):
                 swap(i - 1, i)
                 swapped = True
                 if simulation:
-                    print(arr)
+                    iteration = iteration + 1
+                    print("iteration",iteration,":",arr)
```

Comment:
could you print the array as
```python
print("iteration", iteration, ":", *arr)
```
so that we see the numbers without the `[]`.

Reply:
Changed :) 
arr --> *arr